wiki/Tournaments/CJB/2018/4K/en.md

```diff
@@ -1,16 +1,22 @@
+---
+tags:
+- CJB
+- CJB 2018
+- CJB2018
+---
 The 4th CJB osu!mania 4K Tournament
 =========================
 
-![CJB 4K 4th](banner.png)
+![CJB 4K 4th](img/banner.png)
 
-**The 4th CJB osu!mania 4K Tournament _(CJB 4K 4th)_**, is the fourth one-on-one osu!mania 4K specified Chinese regional tournament hosted by CJB team.
+**The 4th CJB osu!mania 4K Tournament _(CJB 4K 4th)_**, is a one-on-one osu!mania 4K specified Chinese regional tournament hosted by the CJB team. It is the 4th installment of the CJB osu!mania 4K Tournament.
```

Comment:
`**The 4th CJB osu!mania 4K Tournament _(CJB 4K 4th)_**` -> 
`**The 4th CJB osu!mania 4K Tournament** (***CJB 4K 4th***)`

Remove the comma before `is`.